src/Client.js

```diff
@@ -1042,8 +1042,8 @@ class Client extends EventEmitter {
         }
 
         return await this.pupPage.evaluate(async number => {
-            const wid = window.Store.WidFactory.createWid(number);
-            const result = await window.Store.QueryExist(wid);
+            //const wid = window.Store.WidFactory.createWid(number);
+            const result = await window.Store.QueryExist("phone","+"number);
```

Comment:
Sintax error... 
```
const result = await window.Store.QueryExist( "phone" , "+" + number );
```
But with this works for me...

Reply:
**This works for me...**